### PR TITLE
Blender CheatSheet: Updated Syntax

### PR DESCRIPTION
**Pull request by RomainLG**
Am I ok?
____
https://duck.co/ia/view/blender_cheat_sheet
Reference: #1250

### CODE REVIEW

File: share/goodie/cheat_sheets/blender.json [hunk 1]
```diff
@@ -223,56 +223,56 @@
...
             "val": "Outliner",
-            "key": "[Shift] + [F9]"
+            "key": "[Shift] [F9]"
         }, {
             "val": "UV/Image editor",
-            "key": "[Shift] + [F10]"
+            "key": "[Shift] [F10]"
         }, {
             "val": "Text editor",
             "key": "[Shift] + [F11]"
```
Comment (MrChrisW) [inline share/goodie/cheat_sheets/blender.json]: Looks like you've missed this one :+1:

File: share/goodie/cheat_sheets/blender.json [hunk 2]
```diff
@@ -508,35 +508,35 @@
             "key": "k"
         }, {
             "val": "Lock strip",
-            "key": "[Shift] + [l]"
+            "key": "[Shift] [l]"
         }, {
             "val": "Unlock strip",
-            "key": "[Shift] + [Alt] + [l]"
+            "key": "[Shift] [Alt] + [l]"
```
Comment (MrChrisW) [inline share/goodie/cheat_sheets/blender.json]: Looks like you've missed this one :+1:

**Comment by MrChrisW** [comment]
@RomainLG This looks good, few missed `+`, besides those this is fine :+1:

**Comment by RomainLG** [comment]
I think they're all changed now.

**Comment by moollaza** [comment]
@RomainLG thanks a lot for doing this! 

Only problem is that another dev actually made a PR that update this one *just* before you: #1252  Granted, the PR title doesn't mention the other IA's that were fixed.

To make it a little easier for him to merge his PR I'm going to close this one (otherwise he'll run into conflicts on that PR).

Again thanks a lot for doing this, but please make sure you check the existing PR's and leave comments in the related issues so others know your intentions.

**Comment by RomainLG** [comment]
OK no problem :)

**Comment by moollaza** [comment]
>Is there a way to "reserve a PR"? Start the PR when you start working on the file and then push it again when it's finished?

Ideally everyone should be on the same page (feel free to chat with others on Gitter) so that we're not competing for PR :), but you're welcome to submit an unfinished PR and let us know it's a Work In Progress. We'll label it accordingly.

**Pull request closed without merging**